py-polars/tests/unit/utils/test_utils.py

```diff
@@ -57,10 +57,15 @@ def test_time_to_pl_time(t: time, expected: int) -> None:
     assert _time_to_pl_time(t) == expected
 
 
-def test_date_to_pl_date() -> None:
-    d = date(1999, 9, 9)
-    out = _date_to_pl_date(d)
-    assert out == 10843
+@pytest.mark.parametrize(
+    ("d", "expected"),
+    [
+        (date(1999, 9, 9), 10843),
+        (date(1969, 12, 31), -1),
```

Comment:
Nice one; good to test across the epoch 👍

Reply:
Thanks - your comment motivated me to add some tests for the min/max boundaries as well 😄 all seems to work well.